packages/medusa/src/services/claim.ts

```diff
@@ -332,9 +332,30 @@ export default class ClaimService extends TransactionBaseService {
             lines as LineItem[]
           )
         }
-
         let newItems: LineItem[] = []
+
         if (isDefined(additional_items)) {
+          await Promise.all(
```

Comment:
todo: I would remove this check. It is generally business users who create claims and they prefer having the control of forcing a claim through despite there not being stock at this time. E.g., if they know that the items will be restocked soon. 

I can see other use cases where merchants might not want this behavior, but would decouple the check from the claim creation and instead look toward a customization option in a future operation.

Reply:
One thing that may make sense to add now is reserving stock for claims. (Would still decouple this from the claim creation and add it in the API handler instead.)

Reply:
that is being handled in the current flow that Phil covered. I believe it is better to have another PR to change the place where the stock is being allocated.
By the way, are we keeping `reserve` or should we rename everything to `allocate`